### DISCUSSION TITLE
Mass mailing

### DISCUSSION THREAD

**Discussion by ZIRTO-PL** [YetiForceCompany/YetiForceCRM#16553] [How to]
Hello,
I have problem because in my yetiforce i cant see mass mailing option i module campaines 
https://yetiforce.com/pl/baza-wiedzy/dokumentacja/dokumentacja-uzytkownika/item/masowa-wysylka-maili-w-ramach-kampanii

**Comment by rskrzypczak** [comment]
You must have a configured SMTP server and an active "Mass emails" action in the profile settings of a given module.

**Comment by ZIRTO-PL** [comment]
How to do active "Mass emails" in module?

**Comment by rskrzypczak** [comment]
Software configuration/Permissions/Profiles
![image](https://user-images.githubusercontent.com/10221999/194040494-18acc632-4db3-4be4-8069-2bbf2e0f9873.png)
Set actions for a given module, e.g. Accounts, Lead, etc. because it is for them that you use mass mailing

**Comment by ZIRTO-PL** [comment]
It is works. Ploblem was in correct configuration SMTP. I should select mail but not smtp

**Comment by ZIRTO-PL** [comment]
Mam poprawnie skonfigurowany SMTP i niby maile się wysyłają. Niestety jak robię test do siebie, nie dostaję wiadomości.

**Reply by TonyM15** [reply]
if your SMTp is gmail - check All Mails folder

**Reply by ZIRTO-PL** [reply]
It is not gmail. It is my own smtp on cyberfolks

**Comment by Marshal4582** [comment]
do cc somwhere elese.

**Reply by ZIRTO-PL** [reply]
I dont send mails by roundcube but via mass mailing option in module "events"
